File: Data Structures & Algorithms/majority-element-ii/submission-3.java
class Solution {
    public List<Integer> majorityElement(int[] nums) {
        int n = nums.length;
        Map<Integer, Integer> map = new HashMap<>();
        for(int i:nums) map.put(i, map.getOrDefault(i, 0) + 1);
        List<Integer> res = new ArrayList<>();
        for(int i : map.keySet()){
            if(map.get(i) > n / 3) res.add(i);
        }
        return res;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        st = set(nums)
        res = 0
        for x in st:
            if x - 1 in st:
                continue 
            y = x + 1
            while y in st:
                y += 1
            res = max(res, y - x)
        return res